### PR TITLE
Use instance tempfile

### PR DESCRIPTION
**Pull request by cdonati**

### CODE REVIEW

File: lib/remote_helper.py [hunk 1]
```diff
@@ -291,6 +290,39 @@ def is_port_open(cls, host, port, is_verbose):
...
+    Args:
+      host: A str representing the host to enable root logins on.
+      keyname: A str representing the name of the SSH keypair to login with.
+      user: A str representing the name of the user to login as.
+      is_verbose: A bool indicating if we should print the command we execute to
+        enable root login to stdout.
+    """
+    AppScaleLogger.log('Root login not enabled - enabling it now.')
+
+    create_root_keys = 'sudo touch /root/.ssh/authorized_keys'
```
Comment (menivaitsi) [inline lib/remote_helper.py]: What happens if one of these fail?

Comment (menivaitsi) [inline lib/remote_helper.py]: Do you keep going?

Comment (cdonati) [inline lib/remote_helper.py]: No, it will throw an exception and stop.

Comment (menivaitsi) [inline lib/remote_helper.py]: Sounds good.
Please add a "Raises" section to the method docstring to make that clear.

Comment (cdonati) [inline lib/remote_helper.py]: LocalState.shell is the function that raises the exception, though.

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/345/

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/370/

**Pull request merged**